src/components/confirm/confirm.style.js

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+
+const StyledConfirmButtons = styled.div`
+  margin-top: 20px;
+  display: flex;
+  justify-content: flex-end;
+
+  .carbon-button {
```

Comment:
I'm not sure we want to use these class-names here, as they may no longer be on the button.

Reply:
I changed the button to styled button. However, new implementation of Button component is not merged to master yet. I'll add 'DO NOT MERGE' label on this PR until then. Is that okay?